### DISCUSSION TITLE
HintException The user was not created because the user limit has been reached. Check your notifications to learn more.

### DISCUSSION THREAD

**Discussion by Joly0** [nextcloud/all-in-one#5522] [Questions] [answered]
Hey, i am trying to migrate my companies old nextcloud installation to nextcloud-aio.
All went well, but now after all that was done, i noticed the error message `HintException The user was not created because the user limit has been reached. Check your notifications to learn more.` kept appearing in my nextcloud logs. I am just trying to have my previous users from ldap in the new migrated instance.

I have read somewhere else, that here https://nextcloud.com/all-in-one/ it says, that there is a 100 user limit. Also i read somewhere, that this is also enforced though the config.php using a line in the entrypoint.sh file for the nextcloud container that reconfigures this on every startup.

Why is that? Also, i, or my company in this case, would be fine to pay for enterprise support, but the limitation seems artificial and only paying to have more than 100 users in one instance doesnt seem like thats worth it. Especially as we only use nextcloud for the calendar, nothing else. So not worth it at all.

Is there any way to fix this issue and get rid of this limitation other than paying for nextcloud enterprise?

**Comment by szaimen** [comment, marked as answer]
Hi, it was a management decision to add this user limit. Also Github is not the right place to discuss this.
